src/mailhandler.c

```diff
@@ -254,7 +254,7 @@ void sendMail(char* addr, char* body) {
   }
   close(p[0]);
   desc=fdopen(p[1],"w");
-  fprintf(desc,body);
+  fprintf(desc,"%s",body);
```

Comment:
To be honest I can barely write C, even less decent C. If fputs is the same please use that. 

Do you want me to respin a PR or just commit it yourself?